### DISCUSSION TITLE
Mounting Network Drives

### DISCUSSION THREAD

**Discussion by JadenSWang** [photoprism/photoprism#2624] [Q&A]
I can't seem to be able to mount a network smb share, can someone please link me an example yaml file to do such? Following: https://docs.photoprism.app/getting-started/faq/#how-can-i-mount-network-shares-with-docker gives me a services.photoprism.volumes 'type' is a required property

**Comment by graciousgrey** [comment]
This discussion contains an example as far as I remember: https://github.com/photoprism/photoprism/discussions/1041
